### PR TITLE
codify update process for the page

### PR DESCRIPTION
**Pull request by jakubgs**
This adds a `Jenkinsfile` for building the project and pushing to GitHub.
Templates the HTML file for nightlies using `envsubst` using variables defined in `env.groovy`.
Adds na `update_env.sh` for updating links in `env.groovy`.

### CODE REVIEW

**Comment by jakubgs** [comment]
There are two builds:
* https://ci.status.im/job/misc/job/status-im.github.io/ - Builds the actual page using `env.groovy`.
* https://ci.status.im/job/misc/job/status-im.github.io-update_env/ - Updated `env.groovy` based on parameters.

<blockquote><div><strong><a href="https://ci.status.im/job/misc/job/status-im.github.io/">status-im.github.io [misc] [Jenkins]</a></strong></div></blockquote>
<blockquote><div><strong><a href="https://ci.status.im/job/misc/job/status-im.github.io-update_env/">status-im.github.io-update_env [misc] [Jenkins]</a></strong></div></blockquote>

**Comment by jakubgs** [comment]
I still need to add embeding the two new links into the nightlies page.

**Comment by jakubgs** [comment]
It works:
https://ci.status.im/job/misc/job/status-im.github.io-update_env/33

<blockquote><div><strong><a href="https://ci.status.im/job/misc/job/status-im.github.io-update_env/33">misc » status-im.github.io-update_env #33 [Jenkins]</a></strong></div></blockquote>

File: ci/Jenkinsfile.update_env [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    def changes = 0
+    println "params: ${params}"
+    for (param in ['APK_URL', 'IOS_URL', 'DMG_URL', 'OSX_URL']) {
+      println "param: ${param} value: ${params.get(param)}"
+      if (!params.get(param)) {
+        continue /* skip if prameter was not provided */
+      }
+      sh """
+        sed -i -e \\
+          's#env.${param}=.*#env.${param}=\"${params.get(param)}\"#g' \\
```
Comment (adambabik) [inline ci/Jenkinsfile.update_env:25]: It looks like it's duplicated in `update_env.sh`. Would it be possible to avoid that?

**Review by adambabik — APPROVED** [review]

File: ci/Jenkinsfile.update_env (hunk 1 above)
Comment (jakubgs) [reply ci/Jenkinsfile.update_env:25]: Oh yeah, I forgot to remove that script!

Comment (jakubgs) [reply ci/Jenkinsfile.update_env:25]: removed

**Pull request merged**